### DISCUSSION TITLE
Organisations working in domestic home decarbonisation

### DISCUSSION THREAD

**Discussion by SteveHaben** [alan-turing-institute/ADViCE#89] [Key sector: Built Environment]
This thread is to add names of organisations working in the intersection of home decarbonisation and AI. This can include work in areas including heat pumps, district heating, home storage, insulation, retrofitting etc.

**Comment by SteveHaben** [comment]
Below are some organisations working in the area of decarbonising heating with AI, or have worked on innovation projects in this area.

The following organisations produce products for **smart home heating** including preheating, radiator values, and zonal control:
* [Drayton](https://www.draytoncontrols.co.uk/about), part of [Schneider Electric](https://www.se.com/uk/en/home/smart-home/wiser/heating-control/), 
* [Somfy](https://www.somfy.co.uk/help-me-choose/smart-heating)
* [Heatio](https://www.heatio.com/) smart home and energy efficiency recommendations
* [Passiv UK](https://www.passivuk.com/): Smart pre-heating and cooling, and assesses heating system performance.
* [Homely](https://www.homelyenergy.com/): learn household characteristics and preheat to minimise running costs

Smart home/building:
* [Octopus](https://octopusenergy.com/blog/whole-home-flexibility), providing home flexibility and demand side response. 
* [Smappee](https://www.smappee.com/infinity/)
* [Ovo Energy](https://www.ovoenergy.com/smart-home) including smart thermostats
* [Grid Edge](https://gridedge.ai/): smart building management system to flex energy and reduce costs and emissions. 
* [Swarm](https://swarm.eco/)

Using AI to support **heat pump installations** and **operation**:
* [Nesta](https://www.nesta.org.uk/project/boosting-heat-pump-installation-through-generative-ai/), using generative AI chatbot prototype
* Kensa Heat pumps: using AI for [monitoring and predictive maintenance](https://www.kensaheatpumps.com/news-blog/kensa-predict-lower-maintenance-costs-and-longer-heat-pump-lifespans/) 
* [heatly](https://www.heatly.com/) developing [app](https://www.gov.uk/government/publications/heat-pump-ready-programme-successful-projects/heat-pump-ready-programme-stream-2-projects#heatly) for installers to survey, install and commission

Using AI to **upskill**
* [Warmur Academy](https://www.warmuracademy.co.uk/) using AI agent to [help installers upskill](https://ufi.co.uk/voctech-directory/warmur-academy-ai-agent-for-upskilling-heat-pump-installers/)

**Storage** for homes:
* [Sunamp](https://sunamp.com/): Heat and thermal batteries which can be linked to heat pumps, solar PV. 
* [Lunar Energy](https://www.lunarenergy.com/lunar-system), solar-battery system using predictive AI to help scheduling.  

**District heating**
* [City Science](https://cityscience.com/): Using AI to design heat networks, and also home retrofitting. 
* [Guru Systems](https://gurusystems.com/technology/guru-hub-ii/): Use machine learning to pinpoint and resolve issues on heat networks. 
* [Veolia](https://www.veolia.com/en/resources/smart-city/artificial-intelligence-working-our-businesses): monitor and control heating network. 
* [Vattenfall](https://group.vattenfall.com/press-and-media/newsroom/2023/yes-ai-will-improve-energy-efficiency--essentially-all-buildings-can-benefit-from-this-technology#:~:text=AI%20technology%20has%20been%20used,least%20as%20high%20as%20before.)

**Comment by peterbullock** [comment]
Definitely need to add Homely – www.homelyenergy.com – to the list of companies working in decarbonising heating with smart tech. Algorithms learn household characteristics and preheat according to weather forecasts and tariff to minimise running costs.

**Reply by SteveHaben** [reply]
Added!
